src/homework_5/LoggerApp.java

```diff
@@ -0,0 +1,22 @@
+package homework_5;
+
+import java.util.logging.*;
```

Comment:
doesn't use auto import
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad